lib/components/dashboard/Navigation/projectHeader.tsx

```diff
@@ -0,0 +1,19 @@
+import {Project} from '@prisma/client'
+import {useEffect, useState} from 'react'
+import { getProject } from './actions'
+import { Skeleton } from '~/components/ui/skeleton'
+
+type Props = {
+	projectId: string
+}
+
+export default function ProjectRoute({projectId}: Props) {
+	const [project, setProject] = useState<Project>()
+	useEffect(() => {
+		async function setProj() {
+			setProject(await getProject(projectId))
+		}
+		setProj()
```

Comment:
Could this be done server-side @adamgobes?

Reply:
Was gonna do that, but the entire nav bar is client-side. We can't put server components inside the client so that's the reason for this.